src/modules/DomainModule.php

```diff
@@ -639,6 +639,14 @@ public function domainSetWhoisProtect($row, $enable = null)
             return $row;
         }
 
+        if (in_array($this->getDomainTopZone($row['domain']), $this->tool->getDisabledWP(), true)) {
+            return $row;
+        }
+
+        if (in_array($this->getDomainTopZone($row['domain']), ['es'], true)) {
+            return $row;
+        }
+
```

Comment:
можно объединить

```suggestion
        if ($this->isDisabledWPZone($row['domain'])) {
            return $row;
        }
```

и добавить соответствующую функцию